Assets/Scripts/Gameplay/GameState/ServerBossRoomState.cs

```diff
@@ -85,14 +95,37 @@ public void OnServerLoadComplete(SceneEvent sceneEvent)
             }
         }
 
+        private void ClientLoaded(ulong clientId, string sceneName)
```

Comment:
@LPLafontaineB this would conflict with your changes with connection state right? you change the way spawn is handled

Reply:
Yes it will, I'll just have to look into it after this is merged